I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 458, in script
    "Воспользуйтесь удочкой"
  File "game/inventory.rpy", line 1, in execute
    screen inventory:
  File "game/inventory.rpy", line 1, in execute
    screen inventory:
  File "game/inventory.rpy", line 4, in execute
    frame:
  File "game/inventory.rpy", line 11, in execute
    vbox:
  File "game/inventory.rpy", line 13, in execute
    if invent:
  File "game/inventory.rpy", line 15, in execute
    hbox:
  File "game/inventory.rpy", line 19, in execute
    viewport id "box":
  File "game/inventory.rpy", line 25, in execute
    vbox:
  File "game/inventory.rpy", line 26, in execute
    for i in range(0, len(items)):
  File "game/inventory.rpy", line 27, in execute
    imagebutton:
  File "game/inventory.rpy", line 27, in keywords
    imagebutton:
  File "game/inventory.rpy", line 28, in <module>
    idle Image(GetFN(i))
  File "game/initpython.rpy", line 35, in GetFN
    fn, hn = items[index]
ValueError: too many values to unpack

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 458, in script
    "Воспользуйтесь удочкой"
  File "renpy/ast.py", line 721, in execute
    renpy.exports.say(who, what, *args, **kwargs)
  File "renpy/exports.py", line 1419, in say
    who(what, *args, **kwargs)
  File "renpy/character.py", line 1249, in __call__
    self.do_display(who, what, cb_args=self.cb_args, dtt=dtt, **display_args)
  File "renpy/character.py", line 901, in do_display
    **display_args)
  File "renpy/character.py", line 647, in display_say
    rv = renpy.ui.interact(mouse='say', type=type, roll_forward=roll_forward)
  File "renpy/ui.py", line 298, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "renpy/display/core.py", line 3325, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, **kwargs)
  File "renpy/display/core.py", line 3737, in interact_core
    root_widget.visit_all(lambda i : i.per_interact())
  File "renpy/display/core.py", line 568, in visit_all
    d.visit_all(callback, seen)
  File "renpy/display/core.py", line 568, in visit_all
    d.visit_all(callback, seen)
  File "renpy/display/core.py", line 568, in visit_all
    d.visit_all(callback, seen)
  File "renpy/display/screen.py", line 436, in visit_all
    callback(self)
  File "renpy/display/core.py", line 3737, in <lambda>
    root_widget.visit_all(lambda i : i.per_interact())
  File "renpy/display/screen.py", line 447, in per_interact
    self.update()
  File "renpy/display/screen.py", line 637, in update
    self.screen.function(**self.scope)
  File "game/inventory.rpy", line 1, in execute
    screen inventory:
  File "game/inventory.rpy", line 1, in execute
    screen inventory:
  File "game/inventory.rpy", line 4, in execute
    frame:
  File "game/inventory.rpy", line 11, in execute
    vbox:
  File "game/inventory.rpy", line 13, in execute
    if invent:
  File "game/inventory.rpy", line 15, in execute
    hbox:
  File "game/inventory.rpy", line 19, in execute
    viewport id "box":
  File "game/inventory.rpy", line 25, in execute
    vbox:
  File "game/inventory.rpy", line 26, in execute
    for i in range(0, len(items)):
  File "game/inventory.rpy", line 27, in execute
    imagebutton:
  File "game/inventory.rpy", line 27, in keywords
    imagebutton:
  File "game/inventory.rpy", line 28, in <module>
    idle Image(GetFN(i))
  File "game/initpython.rpy", line 35, in GetFN
    fn, hn = items[index]
ValueError: too many values to unpack

Windows-10-10.0.19041
Ren'Py 7.4.11.2266
Diplom1 1.0
Mon Jun 20 00:22:10 2022
